### ISSUE TITLE
build_pod.sh  create symlink maybe  should use `ln -fs` instead of `ln -Fs`

### ISSUE THREAD

**Issue by boyan01** [superlistapp/super_native_extensions#211]
After introducing super_clipboard into my project, I always get an error when building.

```
ln: failed to create symbolic link 'project_dir/build/macos/Build/Products/Profile/super_native_extensions/cargokit_phony': File exists
```

os version: silicon macOS 13.5

It seems that caused by `cargokit/build_pod.sh`.

https://github.com/superlistapp/super_native_extensions/blob/7bec77b5c784f1ba4d1a1819923247c306c600dc/super_native_extensions/cargokit/build_pod.sh#L54-L58

according `ln --help`, I guess this is type mistake.

```
  -d, -F, --directory         allow the superuser to attempt to hard link
                                directories (note: will probably fail due to
                                system restrictions, even for the superuser)
  -f, --force                 remove existing destination files
```

**Comment by knopp** [comment]
Ouch. That definitely should have been lowercase f.

**Comment by knopp** [comment]
Should be fixed in 0.7.0-dev.7. Please reopen if the problem exists.
